src/utils/dom.ts

```diff
@@ -162,15 +162,28 @@ export async function focusElement(el: FocusableElement): Promise<void> {
   return isCalciteFocusable(el) ? el.setFocus() : el.focus();
 }
 
+function getAssignedElements<T extends Element = Element>(element: Element, slotName: string | string[]): T[] {
+  const slotSelector = Array.isArray(slotName)
+    ? slotName.map((name) => `slot[name="${name}"]`).join(",")
+    : slotName
+    ? `slot[name="${slotName}"]`
+    : "slot:not([name])";
+
+  const slots = Array.from(element.shadowRoot.querySelectorAll(slotSelector)) as HTMLSlotElement[];
+
+  const elements = slots
+    .map((slot) => slot.assignedElements({ flatten: true }) as T[])
+    .reduce((previousValue, currentValue) => [...previousValue, ...currentValue], []);
+
+  return elements;
+}
+
 interface GetSlottedOptions {
   all?: boolean;
-  direct?: boolean;
```

Comment:
Do we need direct? If we use assignedElements() then we can be sure that it is direct. Thats a benefit IMO.